### ISSUE TITLE
Home, End, PageUp, and PageDown don't seem to work w/ visual mode

### ISSUE THREAD

**Issue by chandlerc** [lunixbochs/ActualVim#157]
When creating a visual selection, hitting any of the above keys causes ActualVim to exect the visual mode entirely. Should be pretty easy to reproduce. Seeing this w/ near-master ActualVim / NeoVim.

Side note, is there a way to donate to support the development of ActualVim? I continue to find ST3 + ActualVim to be the best GUI Vim setup I can find and while I have limited time to contribute patches, I'd love to contribute in more ways than an endless stream of bug reports. ;]

**Comment by lunixbochs** [comment]
Hmm, I wonder if those keys are even being sent to vim. If not, should be an easy fix.

Donations are greatly appreciated. I have a Patreon here that talks more about my situation: https://www.patreon.com/lunixbochs

**Comment by lunixbochs** [comment]
Yep, easy fix! The keymap is just missing those keys: https://github.com/lunixbochs/ActualVim/blob/master/Default.sublime-keymap

**Comment by lunixbochs** [comment]
Thanks for the support! Missing key bindings be fixed in master, can you verify?

**Comment by chandlerc** [comment]
Verified that it works! Awesome!

I noticed an odd issue with the 'end' key though. When making a visual selection, if I hit "end", the cursor ends up at the beginning of the next line. Easy to see by going to the last column of text in normal mode. Hitting 'end' does nothing. Go to insert, hit end, it goes to insert one-past-the-end. Hit escape and go to visual mode and hit 'end' and it tries to go to one-past-the-end, but where neovim and vim show the block cursor past the end of the current line (like the bar cursor for insert), ST3 shows the cursor at the start of the next line.

Happy to move this to a separate issue if more appropriate.
